applications/visualizer/backend/visualizer/settings/development.py

```diff
@@ -3,6 +3,8 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
+GCS_BUCKET_URL = BASE_DIR.parent.parent.parent / "data"
```

Comment:
suggestion (non-blocking): I believe here you meant to use DATA_FOLDER_FRAGMENT instead of "data"

Reply:
Good catch, I actually need to remove the `DATA_FOLDER_FRAGMENT` instead. It's used only here, and it doesn't help understanding better where or what it really represent without context.